examples/lib/shops/spiders/allo_ua.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Spiders.AlloUa do
+  use Crawly.Spider
+
+  def override_settings() do
+    ui_node = System.get_env("UI_NODE") || "ui@127.0.0.1"
+    ui_node = ui_node |> String.to_atom()
+
+    pipelines = [
+      {Crawly.Pipelines.Validate, fields: [:title, :description, :price]},
+      {Crawly.Pipelines.DuplicatesFilter, item_id: :title},
+      {Crawly.Pipelines.Experimental.SendToUI, ui_node: ui_node},
+      Crawly.Pipelines.JSONEncoder,
+      {Crawly.Pipelines.WriteToFile, extension: "jl", folder: "/tmp"}
+    ]
+
+    [
+      concurrent_requests_per_domain: 2,
+      closespider_itemcount: 1000,
+      pipelines: pipelines,
+      closespider_timeout: 1
+    ]
+  end
+
+  @impl Crawly.Spider
+  def base_url(), do: "https://allo.ua/ru/"
+
+  @impl Crawly.Spider
+  def init() do
+    [
+      start_urls: [
+        "https://allo.ua/ru/products/notebooks/"
+      ]
+    ]
+  end
+
+  @impl Crawly.Spider
+  def parse_item(response) do
+    # Parse response body to document
+    {:ok, document} = Floki.parse_document(response.body)
+
+    # Extract individual product page URLs
+    urls =
```

Comment:
Could you also extract links from pagination here?

Reply:
added